sigle/src/modules/analytics/NftLockedView.tsx

```diff
@@ -0,0 +1,85 @@
+import Image from 'next/image';
+import { sigleConfig } from '../../config';
+import { Box, Button, Flex, Heading, Text, Typography } from '../../ui';
+import { DashboardLayout } from '../layout';
+
+export const NftLockedView = () => {
+  return (
+    <DashboardLayout>
+      <Box
+        css={{
+          display: 'grid',
+          placeItems: 'center',
+
+          '@xl': {
+            height: '100%',
+          },
+        }}
+      >
+        <Flex gap="5" direction="column" align="center">
+          <Box css={{ position: 'relative' }}>
+            <Image width={144} height={131} src="/static/img/nft_locked.gif" />
+          </Box>
+          <Typography as="h3" size="h3">
+            ✨ Creator plan ✨
+          </Typography>
+          <Typography css={{ textAlign: 'center' }} size="subheading">
+            Do more with Sigle and access premium features for your Web3 blog
+          </Typography>
+          <Flex
+            as="ul"
+            css={{
+              backgroundColor: '$green2',
+              br: '$2',
+              color: '$green12',
+              py: '$5',
+              px: '$10',
+            }}
+            direction="column"
+            gap="3"
+          >
+            <Typography size="subheading" as="li">
+              {' '}
```

Comment:
is this normal?

Reply:
hmm yeah shouldn't be there so will update :)